libs/external-db-mysql/src/index.ts

```diff
@@ -0,0 +1,29 @@
+export {default as SchemaProvider} from './mysql_schema_provider'
```

Comment:
can you add space between the brackets? (one after and one before) like you did in line 2 for example